src/longbow/segment/command.py

```diff
@@ -526,11 +537,14 @@ def _write_segmented_read(
     :param delimiters: A list of tuples containing the names of delimiter sequences to use to split the given read.
     :param bam_out: An open pysam.AlignmentFile ready to write out data.
     :param barcode_conf_file: An open file ready to write out the barcodes and confidence scores.
+    :param ignore_cbc_and_umi: Boolean indicating whether we should ignore CBC/UMI tags.
+    :param read_index: The index of the read.
     :return: the number of segments written.
     """
 
     segment_bounds_tuples = segment_read_with_simple_splitting(read, delimiters, segments)
 
+    split_read_index = read_index * 100
```

Comment:
I think we should make this dependent on the number of segments in the model.  If we somehow had a lot of model segments, 100 might not be big enough to disambiguate them all (it's good for the 15 element ones).

Reply:
Good idea - will change.